pkg/console/console.go

```diff
@@ -49,6 +50,7 @@ type FrontendConfig struct {
 	Language    string `json:"language" name:"-"`
 	SupportLink string `json:"support_link" name:"support-link" description:"The URI that the support button will point to"`
 	StackConfig `json:"stack_config" name:",squash"`
+	SentryLink  config.Sentry `json:"sentry_link" name:"sentry-link" description:"The Sentry URI used for error reporting"`
```

Comment:
1. This is shared between frontend apps, right? Then it belongs in `webui.TemplateData`.
2. Just call it `SentryDSN` instead of including the `config.Sentry` type
3. Please keep the description consistent.